alacritty/src/renderer/mod.rs

```diff
@@ -420,6 +423,27 @@ struct InstanceData {
     bg_a: f32,
 }
 
+#[derive(Copy, Debug, Clone)]
+pub enum LayerType {
+    RegularGlyphs = 0,
+    ColoredGlyphs = 1,
+}
+
+#[derive(Debug)]
+pub struct Layer {
+    current_atlas: usize,
+    active_tex: GLuint,
+    batch: Batch,
+    atlas: Vec<Atlas>,
+}
+
+impl Layer {
+    #[inline]
+    pub fn new() -> Self {
+        Self { current_atlas: 0, active_tex: 0, batch: Batch::new(), atlas: vec![] }
+    }
+}
```

Comment:
This looks like it could be replaced with a default implementation (should auto-derive)?

Could still have new, but that should probably then just call default.